assets/js/implicit-login.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener( 'DOMContentLoaded', function() {
+    if ( ! window.location.hash || window.location.hash.indexOf('id_token') < 0 ) {
+        return;
```

Comment:
If the token is missing you return. Does that mean that you are ignoring a possible auth error?

Reply:
Good question and I don't know if I have a good answer. The core login page processing does not use anchors at all and I don't want this to run unless I have something to do here. I'm not sure of another situation where we would have a hash and want to do something with it. I don't see anything else in the [implicit grant doc](https://auth0.com/docs/api-auth/tutorials/implicit-grant). 

Is there an error message that I might get back? `error` or `error_description`?